level2/instance.tf

```diff
@@ -66,13 +65,16 @@ resource "aws_security_group" "public" {
 
 
 resource "aws_instance" "private" {
+  count = 2
+
   ami                    = data.aws_ami.amazonlinux.id
   instance_type          = "t3.micro"
   key_name               = "devops"
   vpc_security_group_ids = [aws_security_group.private.id]
-  subnet_id              = data.terraform_remote_state.level1.outputs.private_subnet_id[0]
+  subnet_id              = data.terraform_remote_state.level1.outputs.private_subnet_id[count.index]
+  user_data              = file("user-data.sh")
   tags = {
-    Name = "${var.env_code}-public"
+    Name = "${var.env_code}-private"
```

Comment:
Change this also `Name = "${var.env_code}-private"` to  `Name = "${var.env_code}-private${count.index}"`